homeassistant/components/seventeentrack/sensor.py

```diff
@@ -77,32 +49,43 @@ async def async_setup_platform(
     async_add_entities: AddEntitiesCallback,
     discovery_info: DiscoveryInfoType | None = None,
 ) -> None:
-    """Configure the platform and add the sensors."""
-
-    session = aiohttp_client.async_get_clientsession(hass)
-
-    client = SeventeenTrackClient(session=session)
+    """Initialize 17Track import from config."""
+    async_create_issue(
+        hass,
+        HOMEASSISTANT_DOMAIN,
+        f"deprecated_yaml_{DOMAIN}",
+        is_fixable=False,
+        breaks_in_ha_version="2024.7.0",
+        severity=IssueSeverity.WARNING,
+        translation_key="deprecated_yaml",
+        translation_placeholders={
+            "domain": DOMAIN,
+            "integration_title": "17Track",
+        },
+    )
```

Comment:
Issues should be guiding the user more to what they did wrong. Did the connection fail? Are their credentials incorrect? Please check integrations like `lutron`, `suez_water` or `streamlabswater` for examples

Reply:
updated